src/dialog/index.ts

```diff
@@ -363,12 +413,27 @@ class Dialog extends Panel {
    *
    * @param event - The DOM event sent to the widget
    */
-  protected evtContextMenu(event: Event): void {
+  protected _evtContextMenu(event: Event): void {
     event.preventDefault();
     event.stopPropagation();
   }
 
   /**
+   * Find the first focusable item in the dialog.
+   */
+  private _findFirst(): void {
+    let candidateSelectors = [
+      'input',
+      'select',
+      'a[href]',
+      'textarea',
+      'button',
+      '[tabindex]',
+    ].join(',');
+
+    this._first = this.node.querySelectorAll(candidateSelectors)[0] as HTMLElement;
```

Comment:
are we always guaranteed to find something that fits this criteria? In other words, might `this.node.querySelectorAll(candidateSelectors)` ever return nothing?